README.md

```diff
@@ -35,8 +35,8 @@ regarding Chainlink social accounts, news, and networking.
 
 1. [Install Go 1.17](https://golang.org/doc/install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
    - Example Path for macOS `export PATH=$GOPATH/bin:$PATH` & `export GOPATH=/Users/$USER/go`
-2. Install [NodeJS 12.22](https://nodejs.org/en/download/package-manager/) & [Yarn](https://yarnpkg.com/lang/en/docs/install/)
-   - It might be easier long term to use [nvm](https://nodejs.org/en/download/package-manager/#nvm) to switch between node versions for different projects: `nvm install 12.22 && nvm use 12.22`
+2. Install [NodeJS](https://nodejs.org/en/download/package-manager/) & [Yarn](https://yarnpkg.com/lang/en/docs/install/). See the current version in `package.json` at the root of this repo under the `engines.node` key.
+   - It might be easier long term to use [nvm](https://nodejs.org/en/download/package-manager/#nvm) to switch between node versions for different projects. For example, assuming $NODE_VERSION was set to a valid version of NodeJS, you could run: `nvm install $NODE_VERSION && nvm use $NODE_VERSION`
```

Comment:
IMHO we should push the usage of NIX or [asdf](https://asdf-vm.com/) as an alternative to `nvm`. [.tool-versions](https://github.com/smartcontractkit/chainlink/blob/develop/.tool-versions) is already configured.

Reply:
I personally use `asdf` so I'm a fan but it looks like we have an `.nvrmc` file too which specifies the correct version as well and others might be relying on that.